pkg/apis/core/v1beta1/defaults/defaults.go

```diff
@@ -87,10 +90,20 @@ func SetDefaultKubeFedConfig(fedConfig *v1beta1.KubeFedConfig) {
 		spec.SyncController = &v1beta1.SyncControllerConfig{}
 	}
 
+	syncController := spec.SyncController
+	setInt64(&syncController.MaxConcurrentReconciles, DefaultSyncControllerMaxConcurrentReconciles)
```

Comment:
nit: no need to introduce variable here:

```suggestion
	setInt64(&spec.SyncController.MaxConcurrentReconciles, DefaultSyncControllerMaxConcurrentReconciles)
```